mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -192,11 +200,14 @@ def __init__(self,
             head_conv=head_conv,
             final_kernel=final_kernel,
             bias=bias)
-
-    def init_weights(self):
-        """Initialize weights."""
-        self.cls_head[-1].bias.data.fill_(self.init_bias)
-        self.task_head.init_weights()
+        if init_cfg is None:
+            self.init_cfg = dict(
+                type='Kaiming',
+                layer='Conv2d',
+                override=dict(
+                    type='Kaiming',
+                    name=self.cls_head[-1],
```

Comment:
the value of ``name`` is attribute name. I suggest to rewrite ``init_weights`` for initializing cls_head.